README.rst

```diff
@@ -20,70 +21,114 @@ Khan Academy's exercise question editor and renderer.
 
 See https://github.com/Khan/perseus for more info.
 
+
 What is this plugin?
 --------------------
 
 A Perseus renderer wrapper for Kolibri that can track learning progress and save to the database.
 
-How can I install this plugin?
-------------------------------
 
-1. Inside your Kolibri virtual environment:
-    ``pip install kolibri-perseus-exercise-plugin``
+Installation
+------------
+
+**This plugin is bundled with Kolibri**. If you are running a custom version,
+you can install it like this:
+
 
-2. Activate the plugin:
+1. Inside your Kolibri virtual environment::
 
-    ``kolibri plugin exercise_perseus_renderer enable``
+    pip install kolibri-perseus-exercise-plugin
+
+2. Activate the plugin::
+
+    kolibri plugin exercise_perseus_renderer enable
 
 3. Restart Kolibri.
 
-How can I install this plugin for development?
-------------------------------
 
-1. Download this repo.
+Development guide
+-----------------
+
+1. Clone this repo.
 
 2. Open terminal in your Kolibri repo.
 
-3. run the following commands:
+3. Run the following commands, with your Kolibri development virtual env
+   enabled::
 
-    ``pip install -e <KOLIBRI-PERSEUS-PLUGIN-LOCAL-PATH>``
+    pip install -e <KOLIBRI-PERSEUS-PLUGIN-LOCAL-PATH>
 
-    ``kolibri plugin kolibri_exercise_perseus_plugin enable``
+    kolibri plugin kolibri_exercise_perseus_plugin enable
 
-4. Then run the commands to install frontend packages in Kolibri, this plugin will have its dependencies recursively installed:
+4. Then run the commands to install frontend packages in Kolibri, this plugin
+   will have its dependencies recursively installed::
 
-    ``yarn install``
+    yarn install
 
-5. Finally, to copy over Mathjax into the static folder, run the following command (you will need to do this if you update the version of Perseus in the repo also):
+   Make sure that this message **DOES NOT** appear in your logging output of
+   ``yarn install``, it means that you are not using the development version::
+
+    WARNING  assets/src/module.js not found for plugin exercise_perseus_render_module.
+
+5. After cloning for the first time, or if you made any changes to the project,
+   you need to rebuild it::
```

Comment:
This might be a misleading description? Perhaps 'you need to recopy the static assets from the perseus repo'?